.github/CODEOWNERS

```diff
@@ -1,2 +1,3 @@
 ## code changes will send PR to following teams/users
 * @whispir/developer-experience
+* TEST12333s
```

Comment:
Got some erroneous code accidentally added to the CODEOWNERS file. Will need to delete this to keep the file working correctly.